File: Rscript/formula.R
imports 'mzkit.formula' from "mzkit.dll";

print("(CH3)3CH" :> scan.formula);
print("H2O" :> scan.formula);
print("(CH3)4C" :> scan.formula);
print("C18H23N4O14P(C9H11N2O8P)5" :> scan.formula);
print("C18H23N4O14P(C9H11N2O8P)n" :> scan.formula(n=5));

File: mzkit/Formula.vb
Imports Microsoft.VisualBasic.CommandLine.Reflection
Imports Microsoft.VisualBasic.Scripting.MetaData
Imports SMRUCC.MassSpectrum.DATA.MetaLib

''' <summary>
''' 
''' </summary>
<Package("mzkit.formula", Category:=APICategories.UtilityTools)>
Module Formula

    <ExportAPI("scan.formula")>
    Public Function ScanFormula(formula As String) As FormulaComposition
        Return FormulaScanner.ScanFormula(formula)
    End Function
End Module
